v2/pkg/astvalidation/operation_validation_test.go

```diff
@@ -4870,7 +4957,7 @@ type __Field {
     deprecationReason: String
 }
 
-"""ValidArguments provided to FieldSelections or Directives and the input fields of an
+"""Arguments provided to FieldSelections or Directives and the input fields of an
```

Comment:
intentional?

Reply:
yeah in a base schema it is just arguments
some old autoreplace